aiida_common_workflows/workflows/relax/bigdft/generator.py

```diff
@@ -108,21 +186,78 @@ def get_builder(
             **kwargs
         )
 
+        builder = self.process_class.get_builder()
+
         if relax_type == RelaxType.ATOMS:
             relaxation_schema = 'relax'
+        elif relax_type == RelaxType.NONE:
+            relaxation_schema = 'relax'
+            builder.relax.perform = orm.Bool(False)
         else:
             raise ValueError('relaxation type `{}` is not supported'.format(relax_type.value))
 
-        builder = self.process_class.get_builder()
-        builder.structure = structure
+        pymatgen_struct = structure.get_pymatgen()
+        ortho_dict = None
+        # for single atom computations, we should actually not perform relaxation
+        if pymatgen_struct.ntypesp <= 1:
```

Comment:
What is the `ntypesp` return of a pymatgen structure? Can't this be directly retrieved from the `StructureData`? Is the comment correct? You mention "single atom" but if I understand correctly `ntypesp` is the number of _types_ of atoms. And why would this rule only apply to structures with at most one atom type?

Reply:
The comment is indeed misleading. This is more in line with the comments in #19 , as some inputs have no need for relaxation to be turned on, it only wastes computing power. This needs to be turned into "disable for symmetric".